README.md

```diff
@@ -266,3 +266,11 @@ a part of an existing mutable rule into a new mutable rule. In order for this ru
 the new rule must be identical to the phrasing of the clause or sentence in the old rule and the new rule must be
 self-contained, meaning it does not require a reference to the old rule to be understood. Similarly the old rule must
 still be self-contained and sensible with part of it removed.
+
+**319** Both players and non-players are allowed to submit a pull request to master on pimotte/nomic, 
+without this pull request containing any changes to any rule's text or the game state. 
```

Comment:
But it does change a rule's text, namely the rule from which a sentence/clause is removed, right?